app/controllers/standup_meeting_groups/joins_controller.rb

```diff
@@ -1,7 +1,8 @@
 class StandupMeetingGroups::JoinsController < ApplicationController
   # POST /standup_meeting_groups/:standup_meeting_group_id/joins.turbo_stream
   def create
-    @standup_meeting_group = StandupMeetingGroup.find(params[:standup_meeting_group_id])
+    @standup_meeting_group = StandupMeetingGroup.includes(:standup_meetings)
```

Comment:
There are a couple of view components that call `standup_meetings` on `@standup_meeting_group`, which would trigger database queries without this `includes`.